usr/lib/common/mech_aes.c

```diff
@@ -1191,6 +1292,189 @@ CK_RV aes_ctr_decrypt_update(STDLL_TokData_t *tokdata,
     }
 }
 
+//
+//
+CK_RV aes_xts_encrypt_update(STDLL_TokData_t *tokdata,
+                             SESSION *sess,
+                             CK_BBOOL length_only,
+                             ENCR_DECR_CONTEXT *ctx,
+                             CK_BYTE *in_data,
```

Comment:
General question: we currently never use 'const' in any of the function headers in mech_aes.c and others. But in_data should be 'const' here.